src/markdeck/wait-for-changes

```diff
@@ -2,7 +2,7 @@
 set -e -E -u
 
 SOURCES="/source/slide*.md $(find /source/assets -type d) /source/include-before-body /source/include-after-body"
-TARGET=/target/index.html
+TARGET="$(mktemp)"
```

Comment:
Why not /target/index.html?
With index.html as target, you also detect changes that happened _before_ wait-for-changes was called.